app/renderer/js/notification/helpers.js

```diff
@@ -83,6 +83,36 @@ function parseReply(reply) {
 		reply = reply.replace(regex, streamMention);
 	});
 
+	window.$.ajaxSetup({async: false});
+
+	window.$.getJSON('/json/users', data => {
```

Comment:
One more minor comment you should deconstruct the `$` variable up top.
`const { $ } = window;`